packages/compass-indexes/src/components/create-index-modal/create-index-modal-header.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+  H3,
+  Body,
+  spacing,
+  css,
+  cx,
+  palette,
+  Link,
+} from '@mongodb-js/compass-components';
+import React from 'react';
+
+const headerStyle = css({
+  padding: spacing[5],
```

Comment:
```suggestion
  padding: spacing[800],
```
nit The single digit values here are deprecated (we'll be updating all of them in https://github.com/mongodb-js/compass/pull/6871 )